frontend/src/services/api/retry.ts

```
@@ -0,0 +1,102 @@
+  
+  return false;
+}
+
+/**
+ * Get or initialize retry state for a request
+ */
+export function getRetryState(config: AxiosRequestConfig): RetryState {
+  if (!retryStateMap.has(config)) {
+    retryStateMap.set(config, { retryCount: 0 });
+  }
+  return retryStateMap.get(config)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)